root/src/main/java/com/example/servingwebcontent/MovieListController.java

```diff
@@ -23,4 +25,28 @@ public String login() {
 	public String register() {
 		return "register";
 	}
+
+	@GetMapping("/test")
+	public String test() {
+		try {
+			Class.forName("com.mysql.cj.jdbc.Driver");
+			//Class.forName("com.mysql.cj.jdbc.Driver").newInstance();
+			//Connection con = DriverManager.getConnection("jdbc:mysql://localhost:8080/movielist","root","moviepass");
+			Connection con = DriverManager.getConnection("jdbc:mysql://localhost/movielist?" + "user=root&password=moviepass");
+
+			Statement stmt = con.createStatement();
+			ResultSet rs = stmt.executeQuery("select * from movies");
+
+			ArrayList<movieObject> movies = new ArrayList<movieObject>();
+			while (rs.next()) {
+				movieObject movie = new movieObject(rs.getString(1), rs.getString(2), rs.getString(3), rs.getString(4), rs.getInt(5), rs.getString(6));
+				movies.add(movie);
+				System.out.println(rs.getString(1) + " " + rs.getString(2) + " " + rs.getString(3) + " " + rs.getString(4) + " " + rs.getInt(5) +  " " + rs.getString(6));
+			}
+			con.close();
+		} catch (Exception e) { System.out.println(e); }
```

Comment:
What exceptions can be thrown here? Exception is the broadest exception that catches everything but not a good practice. You should catch the specific exception instead. Also, learn about the difference between a checked and runtime exception if you don't know them: https://docs.oracle.com/javase/tutorial/essential/exceptions/catchOrDeclare.html